docs/endpoint-shell.md

```diff
@@ -0,0 +1,25 @@
+# Interactive shell for endpoint 
+The Endpoint interactive shell is an debug tool for sending specific command to the device. The interactive shell had integrated some useful commands such as the test tool of radio connection, diagnose commands of internet connection, AT command adapter and the test tool of TA transaction.
```

Comment:
Change "an debug tool" to "a diagnostic tool"

Reply:
Change "for sending specific command" to "capable of executing specific commands"